packages/@tinacms/toolkit/src/packages/react-sidebar/components/Sidebar.tsx

```diff
@@ -75,32 +75,23 @@ interface SidebarProps {
 type displayStates = 'closed' | 'open' | 'fullscreen'
 
 const useFetchCollections = (cms) => {
-  const [info, setInfo] = useState<{
-    loading: boolean
-    error: boolean
-    collections: any[]
-  }>({
-    loading: true,
-    error: false,
-    collections: [],
-  })
+  const [collections, setCollections] = useState<any[]>([])
+  const [loading, setLoading] = useState<boolean>(true)
 
   useEffect(() => {
     const fetchCollections = async () => {
       const response = await cms.api.admin.fetchCollections()
```

Comment:
Should we do a try / catch to handle an error for if this fetch fails?

Reply:
In a desire to get this out quickly, I put error handling outside the scope.  This is only for happy path loading.  I've got another card for adding in try / catch and error handling:
https://github.com/tinacms/tinacms/issues/2600

Reply:
This does set the foundation for that.  I've got a POC branch locally that does it; just don't want to hold up this Quality of Life improvement for that.

Reply:
Makes sense to me!